writer/checkstyle.go

```diff
@@ -0,0 +1,69 @@
+package writer
+
+import (
+	"encoding/xml"
+	"fmt"
+	"io"
+	"sort"
+	"sync"
+
+	"github.com/haya14busa/errorformat"
+	"github.com/haya14busa/go-checkstyle/checkstyle"
+)
+
+// CheckStyle represents checkstyle XML writer. http://checkstyle.sourceforge.net/
+type CheckStyle struct {
+	mu    sync.Mutex
+	files map[string]*checkstyle.File
+	w     io.Writer
+}
+
+func NewCheckStyle(w io.Writer) *CheckStyle {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function NewCheckStyle should have comment or be unexported